app/models/die_mitte/role.rb

```diff
@@ -17,6 +17,8 @@ module DieMitte::Role
 
     self.used_attributes += globalize_attribute_names
 
+    self.merge_excluded_attributes = globalize_attribute_names
```

Comment:
Macht das wirklich was wir wollen? `globalize_attribute_names` sind ja die tatsächlichen Datenbankspalten `label_de`, `label_fr` etc.. Wenn wir die ausschliessen, wie verhält es sich dann beim Mergen wenn ein Label in mehreren Sprachen vorhanden ist? Der von globalize generierte Setter `label=` schaut ja auf die aktuelle Sprache, würde das dann nicht die restlichen Sprachen ignorieren während dem Merge?

Reply:
Für so einen Fall wäre dann wohl ein Test noch spannend, egal für welches Verhalten wir uns entscheiden.